src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -255,19 +257,64 @@ private String get_user_myworkspace_site_json(String userEid, String sessionId)
 			{
 				log.error("Cannot find siteJSON object at index " + i + " within instructorSitesJSONArray " + instructorSitesJSONArray.toString());
 			}
-
-
 		}
 
 		JSONObject sitesJSONObject = new JSONObject();
-		sitesJSONObject.put(JSON_ATTR_SITE_COLLECTION, sitesJSONArray);
+		// sort JSONArray by "title" element
+		sitesJSONObject.put(JSON_ATTR_SITE_COLLECTION, sort(sitesJSONArray, "title", true));
 
 		rv.put("projectsString", sitesJSONObject.toString());
 		rv.put("errorMessage", errorMessage);
 		rv.put("requestUrl", requestUrl);
 		return rv;
 	}
 
+	/**
+	 * function for sorting JSONArray object
+	 * @param jsonArr
+	 * @param sortBy
+	 * @param sortOrder
+	 * @return
+	 */
+	private JSONArray sort(JSONArray jsonArr, String sortBy, boolean sortOrder) {
+		JSONArray sortedJsonArray = new JSONArray();
+
+		List<JSONObject> jsonValues = new ArrayList<JSONObject>();
+		for (int i = 0; i < jsonArr.length(); i++) {
+
+			log.info(jsonArr.getJSONObject(i).toString());
+			jsonValues.add(jsonArr.getJSONObject(i));
+		}
+		final String KEY_NAME = sortBy;
+		final Boolean SORT_ORDER = sortOrder;
+		Collections.sort( jsonValues, new Comparator<JSONObject>() {
+
+			@Override
+			public int compare(JSONObject a, JSONObject b) {
+				String valA = new String();
+				String valB = new String();
+				try {
+					valA = (String) a.get(KEY_NAME);
+					valB = (String) b.get(KEY_NAME);
+
+					log.info("valA=" + valA + " valB=" + valB);
```

Comment:
can the message be more meaningful than `valA, valB`

Reply:
I removed the log line -- it would be very verbose in log file.